docs/sources/configure/integrations/outgoing-webhooks/index.md

```diff
@@ -108,7 +108,7 @@ This setting does not restrict outgoing webhook execution to events from the sel
 
 The type of event that will cause this outgoing webhook to execute. The types of triggers are:
 
-- [Escalation Step](#escalation-step)
+- [Manual or Escalation Step](#escalation-step)
```

Comment:
This was decided to distinguish between event-triggered webhooks (fire, ack, etc) versus manual explicit defined action (an escalation step or a click). Introducing an extra option for manual trigger (which I originally [did](https://github.com/grafana/oncall/pull/4934/commits/2bb8a92971907c1a933e9f1eecece125b587da85)) seemed it could cause some confusion about when the webhook should be available otherwise (kind of related, we have also discussed allowing a webhook to have multiple possible trigger types eventually, to avoid having duplicated entries for the same action in different contexts).

Reply:
I see. Maybe I'm missing the idea here but wouldn't it make a simple use case of triggering a webhook manually unusable? For example: if I want to have a "button" that posts the alert group somewhere (or maybe even runs some automated action), but it actually runs multiple times on every escalation step, I'd probably refrain from using this altogether as a user. IMO if I want a manual webhook it's probably something critical I'm hesitant to automate, so if it runs 10 times automatically that doesn't make sense?

let me know if I'm completely getting this wrong though 😄

Reply:
ok nvm I confused "escalation step" with "status change". I thought escalation step runs on every escalation step in escalation chain. escalation step actually means the "Trigger webhook" escalation step. this makes perfect sense then, sorry for the confusion

Reply:
Just to be sure, the trigger type in this case serves mainly as a filter, so you can only use this (Manual or escalation step) webhook in the context of an escalation chain, or through a manual action. Assigning this trigger type to a webhook is not enough to get the webhook working.

The need for the manual trigger is when there is some human review or preparation required before triggering the webhook (so, not every alert group should perform the action). It is a way to decouple the webhook from the escalation and let a human decide if it should be triggered or not. Eventually the same webhook could be setup as an escalation step in a different configuration (but again, that requires explicitly setting the webhook in the chain, right? In any case there is someone deciding when that trigger should happen).

Hope that makes sense :-)